qemu/tests/cpu_model_negative.py

```diff
@@ -0,0 +1,58 @@
+import re
+import logging
+
+from avocado.utils import process
+
+from virttest import cpu
+from virttest import error_context
+from virttest import utils_qemu
+from virttest import utils_misc
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    Runs CPU negative test:
+
+    1. Launch qemu with improper cpu configuration
+    2. Verify qemu failed to start
+
+    :param test: QEMU test object.
+    :param params: Dictionary with test parameters.
+    :param env: Dictionary with the test environment.
+    """
+
+    if ('CPU_MODEL' in params['wrong_cmd'] and
+            '+avx512_bf16' in params['cpu_model_flags']):
```

Comment:
seems we won't get the flag `avx512_bf16` in `params['cpu_model_flags']`, it is not defined in cfg.

Reply:
@nanliu-r Updated, thanks.